NEWS.md

```diff
@@ -18,6 +18,25 @@ purrr and dplyr:
   dplyr (#217).
 
 
+## pluck()
+
+The plucking mechanism used for indexing into data structures with
+`map()` has been extracted into the function `pluck()`. Plucking is
+often more readable to extract an element buried in a deep data
+structure. Compare this syntax-heavy extraction which reads
+non-linearly:
+
+```
+accessor(x[[1]])$foo
+```
+
+to the equivalent pluck:
+
+```
+pluck(x, 1, accessor, "foo")
```

Comment:
It also occurs to me that we've made `pluck()` super duper general:

```R
x %>% f() %>% g()
# same as 
x %>% pluck(f, g)
```

i.e. it's feel like a full on compose operator. I don't think we should change the name or emphasise this, but it's interesting.